python3-requests.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "python3-package-installation",
+    "buildsystem": "simple",
```

Comment:
how did you generate this? requests module is not present here only mutagen and yt-dlp yet it is named requests.